base_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Basedecorator < Nameable
+  def initialize(nameable)
+    @nameable = nameable
+    super
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As I mentioned in one of the previous comments, I think it would be better if you update this `super` to this `super()` as the current one raises an error while creating a person

Reply:
@tresorsawasawa OPTIONAL ones I always ignore but anyway